airbyte-integrations/singer/exchangeratesapi/source/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM airbyte/base-singer:dev
```

Comment:
another case where we might be able to reduce the amount of boilerplate for a "normal" singer integrartion. again with keeping in mind but no action needed now.